docs/configuring-openshift.md

```diff
@@ -0,0 +1,69 @@
+# Configuring OpenShift
+The following sections describe how Minishift can be used to configure your local OpenShift instance provisioned by the Minishift VM. For details about managing the Minishift VM refer to the [Using Minishift](./using.md) section.
+
+<!-- MarkdownTOC -->
+
+- [Updating OpenShift configuration](#updating-openshift-configuration)
+- [Persistent volumes](#persistent-volumes)
+
+<!-- /MarkdownTOC -->
+
+<a name="updating-openshift-configuration"></a>
+## Updating OpenShift configuration
+
+  Once you have [started](#starting-openshift) OpenShift, you can view and change the master and
```

Comment:
that link cannot be right anymore after the split. Please make sure that all links still work.